src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -66,7 +66,11 @@ module DotNet {
     }
   }
 
-  function invokePossibleInstanceMethodAsync<T>(assemblyName: string | null, methodIdentifier: string, dotNetObjectId: number | null, args: any[]): Promise<T> {
+  function invokePossibleInstanceMethodAsync<T>(assemblyName: string | null, methodIdentifier: string, dotNetObjectId: number | null, ...args: any[]): Promise<T> {
+    if (assemblyName && dotNetObjectId) {
+      throw new Error(`For instance method calls, assemblyName should be null. Received '${assemblyName}'.`) ;
```

Comment:
Isn't the text a bit misleading? You don't know if they are trying to call an instance method or a static one, (based on the check) shouldn't the message reflect that?

Reply:
We effectively have the same check on the server. You're expected to either have an assemblyName + methodIdentifier (so we discover a JSInvocable globally) or a methodIdentifier + dotNetObjectId (so we discover it on the type). 

This is where I copied the text from: https://github.com/aspnet/Extensions/pull/2176/files#diff-5bc9491fb100bce7a35de845877af2b1R141